app/assets/javascripts/components/user/student_outreach.es6.jsx

```diff
@@ -23,27 +23,30 @@ class StudentOutreach extends Component {
       container: {
         display: 'flex',
         flexFlow: 'column',
-        justifyContent: 'center',
-        alignItems: 'center',
         flex: '1',
         padding: '12px',
       },
-      image: {
-        width: '152px',
-        height: '152px',
-        marginBottom: '12px',
-        borderRadius: '50%',
-      },
     };
   }
 
   // --------------------------------------------------
   // Render
   // --------------------------------------------------
+  renderResponsibility() {
+    var student = this.props.student;
```

Comment:
You can just do `var responsibility = this.props.student.responsibility;` and update the accesses below.